administrator/components/com_content/forms/article.xml

```diff
@@ -244,279 +244,254 @@
+					<option	value="1">JSHOW</option>
+				</field>
+
+				<field
+					name="show_publish_date"
+					type="list"
+					label="JGLOBAL_SHOW_PUBLISH_DATE_LABEL"
+					useglobal="true"
 				>
-				<option	value="0">JHIDE</option>
-				<option	value="1">JSHOW</option>
-			</field>
-
-			<field
-					name="option_spacer4"
-					type="spacer"
-					hr="true"
-			/>
-
-			<field
-				name="show_item_navigation"
-				type="list"
-				label="JGLOBAL_SHOW_NAVIGATION_LABEL"
-				useglobal="true"
+					<option	value="0">JHIDE</option>
+					<option	value="1">JSHOW</option>
+				</field>
+			</fieldset>
+			<fieldset name="other" label="COM_CONTENT_OTHER_OPTIONS">
+				<field
+					name="show_item_navigation"
+					type="list"
+					label="JGLOBAL_SHOW_NAVIGATION_LABEL"
+					useglobal="true"
 				>
-				<option value="0">JHIDE</option>
-				<option	value="1">JSHOW</option>
-			</field>
-
-			<field
-				name="show_vote"
-				type="list"
-				label="JGLOBAL_SHOW_VOTE_LABEL"
-				useglobal="true"
+					<option value="0">JHIDE</option>
+					<option	value="1">JSHOW</option>
+				</field>
+
+				<field
+					name="show_vote"
+					type="list"
+					label="JGLOBAL_SHOW_VOTE_LABEL"
+					useglobal="true"
 				>
-				<option value="0">JHIDE</option>
-				<option	value="1">JSHOW</option>
-			</field>
-
-			<field
-				name="show_hits"
-				type="list"
-				label="JGLOBAL_SHOW_HITS_LABEL"
-				useglobal="true"
+					<option value="0">JHIDE</option>
+					<option	value="1">JSHOW</option>
+				</field>
+
+				<field
+					name="show_hits"
+					type="list"
+					label="JGLOBAL_SHOW_HITS_LABEL"
+					useglobal="true"
 				>
-				<option	value="0">JHIDE</option>
-				<option	value="1">JSHOW</option>
-			</field>
-
-			<field
-				name="show_noauth"
-				type="list"
-				label="JGLOBAL_SHOW_UNAUTH_LINKS_LABEL"
-				description="JGLOBAL_SHOW_UNAUTH_LINKS_DESC"
-				useglobal="true"
+					<option	value="0">JHIDE</option>
+					<option	value="1">JSHOW</option>
+				</field>
+
+				<field
+					name="show_noauth"
+					type="list"
+					label="JGLOBAL_SHOW_UNAUTH_LINKS_LABEL"
+					description="JGLOBAL_SHOW_UNAUTH_LINKS_DESC"
+					useglobal="true"
 				>
-				<option value="0">JNO</option>
-				<option value="1">JYES</option>
-			</field>
-
-			<field
-				name="urls_position"
-				type="list"
-				label="COM_CONTENT_FIELD_URLSPOSITION_LABEL"
-				useglobal="true"
+					<option value="0">JNO</option>
+					<option value="1">JYES</option>
+				</field>
+
+				<field
+					name="urls_position"
+					type="list"
+					label="COM_CONTENT_FIELD_URLSPOSITION_LABEL"
+					useglobal="true"
 				>
-				<option value="0">COM_CONTENT_FIELD_OPTION_ABOVE</option>
-				<option value="1">COM_CONTENT_FIELD_OPTION_BELOW</option>
-			</field>
+					<option value="0">COM_CONTENT_FIELD_OPTION_ABOVE</option>
+					<option value="1">COM_CONTENT_FIELD_OPTION_BELOW</option>
+				</field>
+			</fieldset>
+			<fieldset name="global" label="COM_CONTENT_GLOBAL_OPTIONS">
```

Comment:
I dont understand what/why these are grouped with the label "Global Options"
what is global about them?